tool/src/main/java/org/datacommons/tool/WriterPair.java

```diff
@@ -49,7 +49,7 @@ public void close() throws IOException {
   private BufferedWriter newWriter(Args.OutputFileType type) throws IOException {
     String filePath = args.outputFiles.get(type).toString();
     if (csvFile != null) {
-      String fileSuffix = FilenameUtils.removeExtension(csvFile.getName()) + ".csv";
+      String fileSuffix = FilenameUtils.removeExtension(csvFile.getName()) + ".mcf";
```

Comment:
When I made this change I think I also broke genmcf test (https://github.com/datacommonsorg/import/blob/master/tool/src/test/java/org/datacommons/tool/GenMcfTest.java#L70-L76). I'll fix that after this goes in...

Reply:
ok, sounds good